G-licious/NetNutritionViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NetNutritionViewController.swift
+//  G-licious
+//
+//  Created by AppDev on 1/25/16.
+//  Copyright © 2016 Maijid Moujaled. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+class NetNutritionViewController: UIViewController, WKNavigationDelegate {
+
+    private let NET_NUTRITION_URL = NSURL(string: "http://nutrition.grinnell.edu/NetNutrition/1/Mobile/Mobile")!
+    
+    var webView: WKWebView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Initialize / configure the webView
+        
+        webView = WKWebView()
+        webView.navigationDelegate = self
+        
+        // Push the webview down 20 pts so status bar is visible
+        webView.frame = CGRectMake(0, 20, view.frame.width, view.frame.height - 20)
+        webView.navigationDelegate = self
+        
+        view.addSubview(webView)
+        view.layoutIfNeeded()
+        
+        webView.loadRequest(NSURLRequest(URL: NET_NUTRITION_URL))
+    }
+    
+    // Logging for debugging
+    
+    func webView(webView: WKWebView, didFailProvisionalNavigation navigation: WKNavigation!, withError error: NSError) {
```

Comment:
Are these delegate methods required, should we be doing something different than just logging when they're called (also, it's `print` in Swift), or can we just get rid of them?